modules/nf-core/sam2lca/analyze/meta.yml

```
@@ -0,0 +1,61 @@
+name: "sam2lca_analyze"
+description: Calling lowest common ancestors from multi-mapped reads in SAM/BAM/CRAM files
+keywords:
+  - LCA
+  - alignment
+  - bam
+  - metagenomics
+  - Ancestor
+  - multimapper
+tools:
+  - "sam2lca":
+      description: "Lowest Common Ancestor on SAM/BAM/CRAM alignment files"
+      homepage: "https://github.com/maxibor/sam2lca"
+      documentation: "https://sam2lca.readthedocs.io"
+      doi: "10.21105/joss.04360"
+      licence: "['GPL v3']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
+      type: file
+      description: BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+  - bai:
+      type: file
+      description: BAM/CRAM/SAM index
+      pattern: "*.{bam.bai,cram.crai,sam.bai}"
+  - database:
+      type: file
+      description: Directory containing the sam2lca database
+      pattern: "*"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - csv:
+      type: file
+      description: CSV file containing the sam2lca results
+      pattern: "*.sam2lca.csv"
+  - json:
+      type: file
+      description: JSON file containing the sam2lca results
+      pattern: "*.sam2lca.json"
+  - bam:
+      type: file
+      description: Optional sorted BAM/CRAM/SAM file annotated with LCA taxonomic informations
+      pattern: "*.sam2lca.bam"
```

```suggestion
      pattern: "*.bam"
```

If you fear a name clash,

https://nf-co.re/docs/contributing/modules#naming-conventions para 6